### ISSUE TITLE
Incomplete WAI-ARIA role description for `tabpanel`

### ISSUE THREAD

**Issue by CuriousPhysicist** [mdn/sprints#3464]
# Request type

- [ ] Please close this issue, I accidentally submitted it without adding any details
- [ ] New documentation
- [x] Correction or update

# Details

While reading the summary page on WAI-ARIA roles I noticed that the `tabpanel` role had an incomplete description. 

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/ARIA:_tabpanel_role

This role is referenced in the `tab` role, where `tablist` is also referenced, but the link is coloured red. I assume from this that the links are broken. 

https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-2/tabs.html

It seems the official definition is available here:

https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-2/tabs.html

The MDN reference docs are a great resource, I'm happy to put together a pull request if it would be helpful.

**Comment by CuriousPhysicist** [comment]
_NB: I have joined the MDN site so should be able to submit a wiki update for review._

**Comment by Elchi3** [comment]
> NB: I have joined the MDN site so should be able to submit a wiki update for review.

Have you been able to edit the wiki?

**Comment by CuriousPhysicist** [comment]
I took a look and reached out to one of the previous authors. I will look again and put something in for review.

Not sure if reference to the tabpanel role should be removed or just better linked to other pages/info.
